contrib/kava-3/upgrade-instructions.md

```diff
@@ -0,0 +1,15 @@
+# Upgrade to kava-3
```

Comment:
You can include this on the repo wiki 👍

Reply:
yeah I think in the repo isn't the place for it